code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -178,6 +196,78 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 49;
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
+pub struct Router {}
+
+impl ModuleRouter for Router {
+	fn get_route_mut(&mut self, _module_id: &ModuleId) -> Option<&mut dyn Module> {
+		None
+	}
+
+	fn has_route(_module_id: &ModuleId) -> bool {
+		false
+	}
+
+	fn lookup_module_by_port(_port_id: &PortId) -> Option<ModuleId> {
+		None
+	}
+}
```

Comment:
Perhaps move this to pallet-ibc and rename to NoneRouter/NeverRouter?

Reply:
https://github.com/ComposableFi/centauri/pull/195/files